### PR TITLE
Typo Errors Fixed In Docs

### PR DESCRIPTION
**Pull request by rajat123456**

### CODE REVIEW

File: Docs/sp-imp-spec.txt [hunk 1]
```diff
@@ -225,11 +225,11 @@
 
     sql_parse.cc:mysql_execute_command() then uses sp.cc:sp_find() to
     get the sp_head for the procedure (which may have been read from the
-    database or feetched from the in-memory cache) and calls the sp_head's
+    database or fetched from the in-memory cache) and calls the sp_head's
     method execute().
-    Note: It's important that substatements called by the procedure do not
+    Note: It's important that sub statements called by the procedure do not
```
Comment (svoj) [inline Docs/sp-imp-spec.txt]: What's wrong about substatement? Seem to be exactly the right word: https://en.wiktionary.org/wiki/substatement

**Comment by svoj** [comment]
Could you update your branch so that it has only one revision fixing these typos?

File: Docs/sp-imp-spec.txt (hunk 1 above)
Comment (rajat123456) [reply Docs/sp-imp-spec.txt]: Ya, you are right, that's my mistake, it's correct

**Comment by rajat123456** [comment]
Updated the branch

**Comment by svoj** [comment]
There're still 4 commits here and substatement change was not reverted. And source repository seem to be lost.

**Comment by rajat123456** [comment]
Yes, source repository seemed to be lost, shall I make a new pull request?

**Comment by svoj** [comment]
Whatever is easier for you.

**Comment by rajat123456** [comment]
And, then you reference that new pull request with this one.

**Comment by rajat123456** [comment]
I have created the new pull request

**Comment by svoj** [comment]
Merged in #527.

**Pull request closed without merging**